.github/ISSUE_TEMPLATE/design_kit.md

```diff
@@ -0,0 +1,46 @@
+---
+name: "Design kit \U0001F308"
+about: Issues related to the design kit
+title: "[Design kit]: Brief description"
+labels: design kit
+assignees: oliviaflory
```

Comment:
@photodow was this branch off of `master`?

Reply:
Yes I think so, should I take it back and do it straight from the release branch? Sorry!

Reply:
I don't think the files getting pulled in will affect anything, so I'm going to go with this.